lib/galaxy/jobs/runners/condor.py

```diff
@@ -66,7 +66,7 @@ def queue_job( self, job_wrapper ):
         container = None
         universe = query_params.get('universe', None)
         if universe and universe.strip().lower() == 'docker':
-            container = self.find_container( job_wrapper )
+            container = self._find_container( job_wrapper )
```

Comment:
The commit has a different id, maybe you can rebase your branch on upstream/release_16.10 .